### PR TITLE
Update

### PR DESCRIPTION
**Pull request by jduss4**

### CODE REVIEW

File: app/controllers/document_controller.rb [hunk 1]
```diff
@@ -130,6 +130,10 @@ def image_annotation_from_id(unit_no, doc_type, image_id, image_info)
     canvas.label = image_info['label']
     canvas.width = annotation.resource['width']
     canvas.height = annotation.resource['height']
+
+    # Needed for Mirador viewing
+    annotation['on'] = annotation.resource['@id']
```
Comment (jduss4) [inline app/controllers/document_controller.rb]: heck yeah!  I was just adding that myself to some Whitman stuff :)

**Comment by jduss4** [comment]
Were there any other gems that could be updated for this (it has been a while since march, although maybe it's not a big thing since I don't think we can bump rails much more at the moment)

**Comment by techgique** [comment]
I'd like to take a deeper look into updating the Rails version, but I think that can be a separate PR. What do you think?

**Comment by jduss4** [comment]
Agreed, let's make a new issue for that!  Updating something is better than nothing!

**Pull request closed without merging**